packaging/docker/env

```diff
@@ -61,6 +65,8 @@ BEE_CLEF_SIGNER_ENABLE=true
 # BEE_PAYMENT_THRESHOLD=10000000000000
 ## excess debt above payment threshold in BZZ where you disconnect from your peer (default 10000000000000)
 # BEE_PAYMENT_TOLERANCE=10000000000000
+## postage stamp contract address
+# BEE_POSTAGE_STAMP_ADDRESS=
```

Comment:
@ralph-pichler is this needed here? maybe a default value?

Reply:
no default value needed here. the goerli addresses will be detected automatically. this flag and the other one only exists for test blockchain deployments and unsupported networks.